kimchi/src/circuits/expr.rs

```diff
@@ -582,11 +582,11 @@ pub enum FeatureFlag {
 
 impl FeatureFlag {
     fn is_enabled(&self) -> bool {
-        todo!("Handle features")
+        true
     }
 }
 
-#[derive(Copy, Clone, Debug, PartialEq, Eq, Serialize, Deserialize)]
+#[derive(Copy, Clone, Debug, PartialEq, Eq, Serialize, Deserialize, Hash)]
```

Comment:
No longer needed, as we don't cache the unnormalized lagrange evaluations anymore.